ctapipe/tools/merge.py

```diff
@@ -116,126 +30,99 @@ class MergeTool(Tool):
     If no pattern is given, all .h5 files of the given directory will be taken as input.
     """
     input_dir = traits.Path(
-        help="Input dl1-directory",
         default_value=None,
+        help="Input dl1-directory",
```

Comment:
change to:

```python
 help=(
    "directory containing ctapipe data files to merge.  "
    "Use `file_pattern` to set which files are considered. "  
)
```

Reply:
sh / bash have a maximum number of command line arguments, which is not really large, this was the reason for having this.

Reply:
Seems it's 2 million, at least on my machine, which should be plenty... But I remember having issues and having to write a config file for the merge tool